STEERING-COMMITTEE.md

```diff
@@ -0,0 +1,69 @@
+# Istio Steering Committee
+
+The Istio Steering Committee was formed to oversee the administrative aspects of the project, including governance, branding, and marketing. Steering was created to allow the Techincal Oversight Committee to exclusively focus on the technical aspects of the project. 
```

Comment:
`Techincal` is a typo - I would recommend you copy paste in an editor with grammar and spell checking